src/open_inwoner/configurations/bootstrap/kic.py

```diff
@@ -63,6 +64,16 @@ def configure(self):
         organization = settings.OIP_ORGANIZATION or settings.ENVIRONMENT
         org_label = f"Open Inwoner {organization}".strip()
 
+        server_certificate, _ = Certificate.objects.get_or_create(
+            label=settings.KIC_SERVER_CERTIFICATE_LABEL,
+            defaults={
+                "type": settings.KIC_SERVER_CERTIFICATE_TYPE,
+            },
+        )
+
+        with open(settings.KIC_SERVER_CERTIFICATE_PUBLIC_CERTIFICATE) as public_cert:
+            server_certificate.public_certificate.save("kic.crt", public_cert)
```

Comment:
This treats the values as if they're always set, but they aren't listed as required. (this also appears in the other wrappers)